controlplane/src/core/repositories/GraphCompositionRepository.ts

```diff
@@ -126,7 +135,7 @@ export class GraphCompositionRepository {
       createdAt: composition.createdAt.toISOString(),
       isComposable: composition.isComposable || false,
       compositionErrors: composition.compositionErrors || undefined,
-      createdBy: composition.createdBy || undefined,
+      createdBy: composition.createdBy || composition.createdByEmail || undefined,
```

Comment:
At some places we have different props for email and id. I'd favor consistency here and let the view "ui" decide what to use when.

Reply:
In this case we return email not id. Please have a look at the query above. The frontend cannot do this because it wont have access to all org memebrs to do a find.

Reply:
OK